lib/tests/streamlit/runtime/runtime_test.py

```diff
@@ -194,7 +196,9 @@ async def test_shutdown_appsessions_on_stop(self):
                 session_id = self.runtime.create_session(
                     MockSessionClient(), MagicMock()
                 )
-                app_session = self.runtime._get_session_info(session_id).session
+                app_session = self.runtime._session_mgr.get_active_session_info(
+                    session_id
+                ).session
```

Comment:
Is this test still correct? My understanding is that the new stop logic involves shutting down _all_ AppSessions (even those that are inactive). Can we add extend this test to include an assertion that an inactive session gets shut down?